### DISCUSSION TITLE
A concern about Tailwind vs Bootstrap

### DISCUSSION THREAD

**Discussion by luongvanphuc** [tailwindlabs/tailwindcss#6598] [General]
Hi everyone, I'm not sure this is the right place to ask this question. My frontend team in our company is going to replace Bootstrap with Tailwind completely. Howere, there are some of the members in our team raised a concern about Tailwind contributors & community. Here it is:
"The only concern I really have about TailwindCSS is its contributors. Despite it being pretty popular, it seems only one regular contributor and author behind it. Sure-sure Linux has also only Linus as a dictator, but there is also a Linux Foundation with thousands of contributors. Do you know something about the guy who is working on this TailwindCSS? Or it is possible he would just disappear and this framework will repeat the path of Semantic-UI?"

I do have my own ideas to answer that concern, but I still need your feedback on it so that I can have a better and correct picture to convince them and also the company. Thank you so much.

May I have your comment also please @adamwathan 🙇

**Comment by browner12** [comment]
Tailwind is actually backed by a company with employees. Last I remember, Bootstrap was run by Mark Otto, who's full time job is working at Github, including some other people with other full time jobs. They both have a thriving community of contributors.  If it's me, I'm less concerned about the product by an actual company of full time employees making money off the product.

**Reply by luongvanphuc** [reply]
Thanks @browner12 for your info 🙇
